transports/janus_websockets.c

```diff
@@ -1370,7 +1366,7 @@ static int janus_websockets_common_callback(
 			const size_t remaining = lws_remaining_packet_payload(wsi);
 			if(ws_client->incoming == NULL) {
 				JANUS_LOG(LOG_HUGE, "[%s-%p] First fragment: %zu bytes, %zu remaining\n", log_prefix, wsi, len, remaining);
-				ws_client->incoming = g_malloc(len+1);
+				ws_client->incoming = g_malloc0(len+1);
```

Comment:
Also redundant

Reply:
Because of this code:
`ws_client->incoming = g_malloc(len+1);
memcpy(ws_client->incoming, in, len);
**ws_client->incoming[len] = '\0';`**

BTW, I forgot removing the last line.